### ISSUE TITLE
Copy lines or files between branches?

### ISSUE THREAD

**Issue by JudyLipinski** [eamodio/vscode-gitlens#675]
Is there any way to copy lines or files between branches?

**Comment by talmdal** [comment]
I asked the same thing here: [https://github.com/eamodio/vscode-gitlens/issues/647](url),

But what I have found is I can compare a file in a branch or stash with the current working tree. Unless the file doesn't exist in the working tree, the right pane is editable. So what I do is select the lines in the left pane and copy them (ctrl-c) and paste them into the right pane.

**Comment by eamodio** [comment]
@JudyLipinski what @talmdal suggests is about the best option currently. Besides what is requested in https://github.com/eamodio/vscode-gitlens/issues/647 (which needs vscode itself to support), how would you like/expect it to work?

**Comment by JudyLipinski** [comment]
Thanks to you both! Reading #647 that covers what I'd like nicely.  I will also try what @talmdal has said. I tried having my current branch compared to another but however I did it, it wasn't editable.  

I write courses and use branches in a non traditional way, showing students how a demo app or solution progresses.  If I edit a package.json or image or data file, back on one of the first steps, it's a bit of work to add the changes to all 12 or so branches that follow...

I found for entire file changes, I would use my file system to copy files to a temporary directory / holding area, and then just switch branches and overwrite.

For lines I'd throw them in a file and copy paste...
